IM_calculation/IM/Burks_Baker_2013_elastic_inelastic.py

```diff
@@ -68,7 +68,9 @@ def Bilinear_Newmark_withTH(
     # % Interpolate p=-ag*m (linearly)
     p = -ag * m
     tg = np.linspace(0, ag.size - 1, ag.size) * dtg
-    t = np.linspace(0, tg[-1], int(tg[-1] / dt) + 1)  # num of steps increased from tg
+    t = np.linspace(
+        0, tg[-1], int(np.ceil(tg[-1] / dt)) + 1
```

Comment:
The division here sometimes caused poor rounding issue and will cause t and tg to have occasional size difference of 1. np.ceil() prevents the issue.

Reply:
Already done in the latest commit